### PR TITLE
Support for Multiple Proxy Managers

### PR DESCRIPTION
**Pull request by khburdette**
The ability to use multiple Sdl Proxy Managers to spin up and represent multiple SDL apps on the vehicle head unit for any given single mobile application.

### CODE REVIEW

**Comment by khburdette** [comment]
@theresalech This PR is ready for review

File: proposals/NNNN-Multi-Proxy-Managers.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* Possible interactions with apps
+* Possibly discovering other issues
+
+Currently there is no officially supported and documented mechanism for creating multiple SDL apps on a vehicle head unit via a single application on the mobile device that would support the capabilities required for both scenarios mentioned above.
+
+OS Specific:
+* Android
+  * Current behavior on 4.8
+    * Every SdlManager requires a unique service
+    * Every SDL app will have it's own foreground service present in the notifications pull down
```
Comment (theresalech) [inline proposals/NNNN-Multi-Proxy-Managers.md]:
```suggestion
    * Every SDL app will have its own foreground service present in the notifications pull down
```

File: proposals/NNNN-Multi-Proxy-Managers.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  * Regression
+    * 4.5 allows you to start as many SdlProxyALM objects as you want withing a single service. 4.8 only allows you to start 1 SdlManager per service
```
Comment (theresalech) [inline proposals/NNNN-Multi-Proxy-Managers.md]:
```suggestion
    * 4.5 allows you to start as many SdlProxyALM objects as you want within a single service. 4.8 only allows you to start 1 SdlManager per service
```

File: proposals/NNNN-Multi-Proxy-Managers.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    * When using the control channels, a single app can start multiple SDLManagers, but not when using multisession
+    * When using the app on a head unit that doesn't support multisession, everything will work just fine, but when switching to a head unit that supports multisession, only 1 app will appear
+  * Workaround
+    * Modify SDLIAPTransport to only use control channels and not multisession channel
+
+## Proposed solution
+
+Recent mobile library releases for both iOS and Android make use of proxy manager classes that make it easier for developers to interact with the vehicle head unit via SDL.  We are proposing updates and enhancements to the existing SdlManagers and their supporting documentation to include the capability for supporting multiple apps on the vehicle head unit via a single mobile application and mobile library.
+
+OS Specific:
```
Comment (theresalech) [inline proposals/NNNN-Multi-Proxy-Managers.md]: Can you please add technical details for these solutions?

Comment (khburdette) [reply proposals/NNNN-Multi-Proxy-Managers.md]: @theresalech I've accepted the suggested edits and added more detail on the solution

Comment (theresalech) [reply proposals/NNNN-Multi-Proxy-Managers.md]: While the proposal states what the solution should be at a high level, there are no supporting technical details that would make it possible to be discussed.  Could you please provide additional details on how this solution would be accomplished?

**Comment by jordynmackool** [comment]
@yoooriii can you please provide a status on this PR as it has been inactive for a large amount of time?

**Comment by jordynmackool** [comment]
Closing as inactive. Should the author wish to bring this PR under Steering Committee review, the author should notify the Project Maintainer (via admin@smartdevicelink.com), and then the PR can be re-opened.

**Pull request closed without merging**